examples/market/scan.ts

```diff
@@ -1,19 +1,23 @@
 /**
  * This example demonstrates how to scan the market for proposals
- * Let's lear what is the average start price
+ * Lets lear what is the average start price
  */
 import { GolemNetwork, ProposalNew } from "@golem-sdk/golem-js";
+import { pinoPrettyLogger } from "@golem-sdk/pino-logger";
 
-const glm = new GolemNetwork({
-  payment: {
+(async () => {
+  const glm = new GolemNetwork({
+    logger: pinoPrettyLogger({
+      level: "debug",
+    }),
     payment: {
-      network: "holesky",
-      driver: "erc20",
+      payment: {
```

Comment:
do we have to double nest it?

Reply:
This became required after recent merges in the branch. We will remove the double nesting for sure. Probably even next week.